### PR TITLE
Update ru_RU.yaml

### PR DESCRIPTION
**Pull request by seoeaa**
added translation into Russian

### CODE REVIEW

File: i18n/ru_RU.yaml [hunk 1]
```diff
@@ -292,216 +291,215 @@ backend:
...
-        other: a community-specific reason
-      desc:
-        other: This question doesn't meet a community guideline.
-    not_clarity:
-      name:
-        other: needs details or clarity
-      desc:
-        other: This question currently includes multiple questions in one. It should focus on one problem only.
-    looks_ok:
+reason:
```
Comment (shuashuai) [inline i18n/ru_RU.yaml:294]: Indentation error

File: i18n/ru_RU.yaml [hunk 2]
```diff
@@ -292,216 +291,215 @@ backend:
...
-      title:
-        other: "[{{.SiteName}}] Подтвердите новый адрес электронной почты"
-      body:
-        other: "Confirm your new email address for {{.SiteName}} by clicking on the following link:<br>\n<a href='{{.ChangeEmailUrl}}' target='_blank'>{{.ChangeEmailUrl}}</a><br><br>\n\nIf you did not request this change, please ignore this email.\n"
-    new_answer:
-      title:
-        other: "[{{.SiteName}}] {{.DisplayName}} answered your question"
-      body:
-        other: "<a href='{{.AnswerUrl}}'>{{.QuestionTitle}}</a><br><br>\n\n{{.DisplayName}}:<br>\n<blockquote>{{.AnswerSummary}}</blockquote><br>\n<a href='{{.AnswerUrl}}'>View it on {{.SiteName}}</a><br><br>\n\n--<br>\n<small><a href='{{.UnsubscribeUrl}}'>Unsubscribe</a></small>"
+        other: Этот пост требует другой причины, не указанной выше.
```
Comment (shuashuai) [inline i18n/ru_RU.yaml:380]: Indentation error

**Review by shuashuai — CHANGES_REQUESTED** [review]
Thank you very much for your contribution. 

For yaml files, there are strict requirements for indentation. If you change the indentation here, it will cause language display errors. In addition to this, there are also indentation modifications below. In addition, there are modifications to the indentation errors below. I just pointed out two errors.

Our translation work is connected to crowdin. If you are not a developer, we recommend you to go to [crowdin-answer](https://crowdin.com/project/answer/ru) to contribute translation. Here you no longer need to consider the file format and only need to focus on the translation itself.  If you have any questions about translation, feel free to contact me.

We look forward to your contribution.

**Comment by Justman100** [comment]
@shuashuai

Can simply closed

**Comment by seoeaa** [comment]
I'll fix it

File: i18n/ru_RU.yaml (hunk 2 above)
Comment (seoeaa) [reply i18n/ru_RU.yaml]: I fixed everything

**Comment by seoeaa** [comment]
I fixed everything

**Comment by seoeaa** [comment]
checked the file   `import yamllint
import yamllint.config
import yamllint.linter

def lint_yaml(file_path):
    with open(file_path, 'r', encoding='utf-8') as file:
        content = file.read()
    conf = yamllint.config.YamlLintConfig('extends: relaxed')
    issues = yamllint.linter.run(input=content, conf=conf, filepath=file_path)
    for issue in issues:
        print(issue)

# Пример использования
lint_yaml('ru_RU.yaml')`

**Comment by Justman100** [comment]
> checked the file  
> 
> 
> `import yamllint
> import yamllint.config
> import yamllint.linter
> 
> def lint_yaml(file_path):
>     with open(file_path, 'r', encoding='utf-8') as file:
>         content = file.read()
>     conf = yamllint.config.YamlLintConfig('extends: relaxed')
>     issues = yamllint.linter.run(input=content, conf=conf, filepath=file_path)
>     for issue in issues:
>         print(issue)
> lint_yaml('ru_RU.yaml')`

Isn't that managed by Crowdin?

**Comment by seoeaa** [comment]
> Crowdin

no I used Python

**Comment by Justman100** [comment]
> > Crowdin
> 
> no I used Python

You change translations are managed by the Crowdin Project

**Comment by seoeaa** [comment]
> > > Краудин
> > 
> > 
> > нет, я использовал Python
> 
> Вы меняете переводы, которые управляются проектом Crowdin.

then add this file to Crowdin

**Comment by Justman100** [comment]
> > > > Краудин
> > > 
> > > 
> > > нет, я использовал Python
> > 
> > 
> > Вы меняете переводы, которые управляются проектом Crowdin.
> 
> then add this file to Crowdin

What for file??? You only changed existing translations! Which one only does in the editor of the Crowdin project!

**Comment by seoeaa** [comment]
> > > > > Краудин
> > > > 
> > > > 
> > > > нет, я использовал Python
> > > 
> > > 
> > > Вы меняете переводы, которые управляются проектом Crowdin.
> > 
> > 
> > затем добавьте этот файл в Crowdin
> 
> Что за файл??? Вы только изменили существующие переводы! Что только и делается в редакторе проекта Crowdin!

хорошо

**Comment by shuashuai** [comment]
@seoeaa  
I saw that someone has updated the Russian translation content on crowdin. Is this your contribution? If so, we will close this PR and give priority to the translation on crowdin!

**Comment by seoeaa** [comment]
> @seoeaa I saw that someone has updated the Russian translation content on crowdin. Is this your contribution? If so, we will close this PR and give priority to the translation on crowdin!

yes

**Comment by Justman100** [comment]
Than this PR can be closed

**Pull request closed without merging**